src/java.xml.crypto/share/classes/org/jcp/xml/dsig/internal/dom/DOMXPathFilter2Transform.java

```diff
@@ -107,7 +107,7 @@ private void unmarshalParams(Element curXPathElem) throws MarshalException
             if (attributes != null) {
                 int length = attributes.getLength();
                 Map<String, String> namespaceMap =
-                    new HashMap<>(length);
+                        HashMap.newHashMap(length);
```

Comment:
Please check these changes with @seanjmullan before integrating.

Reply:
@seanjmullan 
mirror pr in https://github.com/apache/santuario-xml-security-java/pull/64
jira at https://issues.apache.org/jira/projects/SANTUARIO/issues/SANTUARIO-586?filter=reportedbyme